addons/point_of_sale/models/pos_session.py

```diff
@@ -1787,6 +1787,11 @@ def _get_pos_ui_res_company(self, params):
         else:
             company['country'] = None
 
+        company['account_fiscal_country_id'] = self.env['res.country'].search_read(
+            domain=[('id', '=', company['account_fiscal_country_id'][0])],
+            fields=['code'],
+        )[0]
```

Comment:
FYI: there is always a fiscal_country when loading the POS, because there is this constraint: 
https://github.com/odoo/odoo/blob/3cfe548ee03f96713c7a6d462409be9c825d22e7/addons/point_of_sale/models/pos_config.py#L341-L345